isofit/test/conftest.py

```diff
@@ -0,0 +1,14 @@
+"""``pytest`` configuration"""
+
+
+def pytest_collection_modifyitems(items, config):
```

Comment:
Marked tests can be selected with `$ pytest -m <name>`, however `pytest` does not offer a mechanism for selecting tests that do not have a label. The solution seems to be to quiet add a label during test collection. Any test without a label is assigned an `unlabeled` label at collection time, meaning that unlabeled tests can be selected with `$ pytest -m unlabeled`.